src/cudadecoder/batched-threaded-nnet3-cuda-online-pipeline.cc

```diff
@@ -250,6 +256,8 @@ void BatchedThreadedNnet3CudaOnlinePipeline::DecodeBatch(
         "You need to set --word-symbol-table to use partial hypotheses");
     cuda_decoder_->AllowPartialHypotheses();
   }
+  if (end_points) cuda_decoder_->AllowEndPointing();
```

Comment:
Naming of functions varies between `Endpoint` and `EndPoint`. We generally use "endpoint" as a single word, so the former would be better.
```suggestion
  if (end_points) cuda_decoder_->AllowEndpointing();
```